fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneOnlineIndexingTest.java

```diff
@@ -359,15 +365,17 @@ void luceneOnlineIndexingTestWithAllRecordUpdates() {
                     .setRecordStore(recordStore)
                     .setIndex(index)
                     .setInitialLimit(1)
+                    .setLimit(1)
```

Comment:
This is probably unnecessary given the current implementation.
`setLimit` sets the max number of records it will scan per transaction. Right now it doesn't increase the limit for the number of documents that are being indexed, but if it did index more than one record per transaction, it would not call the config loader after each transaction, and the assertion would break.
Thus, I figured it was worth leaving this change in.